src/Phinx/Wrapper/TextWrapper.php

```diff
@@ -1,127 +1,182 @@
 <?php
-/* Phinx
- *
- * (The MIT license)
- * Copyright (c) 2014 Rob Morgan
- * Copyright (c) 2014 Woody Gilk
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated * documentation files (the "Software"), to
- * deal in the Software without restriction, including without limitation the
- * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
- * sell copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
- * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
- * IN THE SOFTWARE.
- */
-
-namespace Phinx\Wrapper;
-
-use Phinx\Console\PhinxApplication;
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Input\ArrayInput;
-use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\Console\Output\StreamOutput;
-
-/**
- * Phinx text wrapper: a way to run `status`, `migrate`, and `rollback` commands
- * and get the output of the command back as plain text.
- *
- * @author Woody Gilk <woody.gilk@gmail.com>
- */
-class TextWrapper
-{
-    private $app;
-    private $env;
-    private $exit_code;
-
-    /**
-     * @param  Phinx\Console\PhinxApplication $app
-     * @param  String                         $default_env  environment fallback, defaults to "development"
-     * @return void
+    /* Phinx
```

Comment:
Now all this is indented incorrectly... please review your own changes before pushing to Github.